CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## tip
 
+* BUGFIX: fix error when response detected as not a wide series for the instant queries. See [this issue](https://github.com/VictoriaMetrics/victorialogs-datasource/issues/302).
```

Comment:
@dmitryk-dk could you please re-phraze it in a way that is more clear to end user?
For example, `fix error "input data must be a wide series" on Grafana alerting page`. This would make it clear to the user that fix is for alerts. And "input data must be a wide series" mentioning would make it searchable if user googles the error.
Please also try using format [#302](https://github.com/VictoriaMetrics/victorialogs-datasource/issues/302) instead of [this issue](https://github.com/VictoriaMetrics/victorialogs-datasource/issues/302), as it removes constantly repeating `this issue` phrases.